client/src/app/login/login/login.component.scss

```diff
@@ -0,0 +1,147 @@
+:root {
+    --input-padding-x: 1.5rem;
+    --input-padding-y: .75rem;
+  }
+
+  .logo {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+
+  }
+
+  .policy {
+    text-align: center;
+    font-family: "Retro Nouveau";
+    color: #443D3C;
+  }
+
+  p {
+    text-align: center;
+    font-family: "Retro Nouveau";
+    color: #443D3C;
+    
+  }
+  
+  .card-signin {
+    border: 0;
+    border-radius: 1rem;
+    box-shadow: 0 0.5rem 1rem 0 rgba(0, 0, 0, 0.1);
+  }
+  
+  .card-signin .card-title {
+    margin-bottom: 2rem;
+    font-weight: 300;
+    font-size: 1.5rem;
+  }
+  
+  .card-signin .card-body {
+    padding: 2rem;
+  }
+  
+  // .form-signin {
+  //   width: 100%;
+  // }
+  
+  .form-signin .btn {
+    font-family: "Retro Nouveau";
+    font-size: 80%;
+    border-radius: 5rem;
+    letter-spacing: .1rem;
+    font-weight: bold;
+    padding: 1rem;
+    transition: all 0.2s;
+  }
+  
+  // .form-label-group {
```

Comment:
Commented out code that will never be used should be deleted. It makes code harder to maintain by mixing in superfluous text.